requirements.txt

```diff
@@ -8,6 +8,7 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+urllib3>=1.9
```

Comment:
I thought it was used in our code (I greped for urllib3 and found lots of these, so thought we should add it, but my python understanding is still limited, so I probably made a mistake. Will remove from my branch